### ISSUE TITLE
Logging on GameJolt takes forever

### ISSUE THREAD

**Issue by Tricky1975** [Tricky1975/Star-Story#309]
This is (unfortunately) a server issue on GameJolt itself.
Nothing I can do to fix that, it's up to the guys at GameJolt

It's reported to GameJolt as well as issue gamejolt/issue-tracker#438

The reason I posted this in my own issue tracker is to let you guys know I'm aware of it, even though there's nothing I can do about it.

**Comment by Tricky1975** [comment]
This issue is still not fixed, and it's getting a bit on my nerves.
If worst comes to worst, I have no choice but to (temporarily) remove all GameJolt features entirely in the next update, only to put it back once I can confirm everything works the way it should again.

**Comment by Tricky1975** [comment]
I set the deadline for now on Feb 1st. If by then this issue is not fixed I will temporarily remove all API support from the game until everything is fixed.

At my last attempt the game did not take forever to login, yet it did constantly block me out as if I did not type my username/token right. Considering that stuff is stored inside the savegame file any typos are impossible since it used to work in the past, and neither my username nor token were changed since that the new game that lead to this particular savegame was started.
